doc/CONFIGURATION.md

```diff
@@ -78,7 +78,7 @@ The available keys and their accepted values are reported in the table below.
 | tls_cert_file | | String | No | Certificate file for TLS. This file must be owned by either the user running pgagroal or root. |
 | tls_key_file | | String | No | Private key file for TLS. This file must be owned by either the user running pgagroal or root. Additionally permissions must be at least `0640` when owned by root or `0600` otherwise. |
 | tls_ca_file | | String | No | Certificate Authority (CA) file for TLS. This file must be owned by either the user running pgagroal or root.  |
-| libev | `auto` | String | No | Select the [libev](http://software.schmorp.de/pkg/libev.html) backend to use. Valid options: `auto`, `select`, `poll`, `epoll`, `iouring`, `devpoll` and `port` |
+| ev_backend | `epoll` | String | No | Select the event handling backend to use. Valid options: `io_uring`, `epoll`, and `kqueue` |
```

Comment:
`auto` should be default - which will look for `io_uring` first...

Reply:
> `auto` should be default - which will look for `io_uring` first...

agree